### PR TITLE
Add note about empty controller methods

### LINKED ISSUE #90: Add documentation about empty methods in controllers
https://github.com/exercism/v3-website/pull/75#discussion_r492046787

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-website/issues/90.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ All other "normal" routes redirect or render HTML.
 Controllers should be "thin".
 Each action should do at most one thing, retrieve at most one thing, and render/redirect.
 To achieve this, controllers call out to Commands, which contain more complex functionality encapsualted in stand-alone procedures. This is known as the Command Pattern or the Interactor Pattern.
+If an action doesn't need any instance variables, we still prefer adding an empty controller action for it. This allows us to have visibility on what actions have been implemented on that controller.
```
Comment (SleeplessByte) [inline docs/CONTRIBUTING.md]: FWIW, in Rails 6, the empty action is _required_.

Comment (iHiD) [reply docs/CONTRIBUTING.md]: I didn't know that. Thanks 🙂

Comment (iHiD) [reply docs/CONTRIBUTING.md]:
```suggestion
Even if an action doesn't need any instance variables, add an empty controller action for it. This allows us to have visibility on what actions have been implemented on that controller.
```

**Comment by iHiD** [comment]
Thanks! 🙂

**Pull request merged**